src/web/src/pages/_app.tsx

```diff
@@ -44,8 +45,10 @@ const App = ({ Component, pageProps }: AppProps) => {
     <ThemeContext.Provider value={{ theme, themeName: theme.palette.type, toggleTheme }}>
       <ThemeProvider theme={theme}>
         <AuthProvider>
-          <NavBar />
-          <Component {...pageProps} />
+          <SearchProvider>
```

Comment:
Do we want to wrap any component with this provider or only the search page component? I think it'd be best to only use the provider where we need it

Reply:
Good catch, didn't think I could only applied it to the search page. Moved it there, and works the same